module3/lessons/Week5/IntroToBootstrap.md

```diff
@@ -0,0 +1,154 @@
+
+Include this `<link>` tag in the `<head>` of your webpage:
+```html
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+```
+
+Include this `<script>` tag before the closing `</body>` tag of your webpage:
+_Note:_ if you are not using JavaScript, you can skip this step.
+```html
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-geWF76RCwLtnZ8qwWowPQNguL3RmwHVBC9FhGdlKrxdiJJigb/j/68SIy3Te4Bkz" crossorigin="anonymous"></script>
+```
+
+## Containers, Rows, & Columns (Oh My!)
+
+Out the box, Bootstrap is based on a grid design. 
+* _Everything_ is built in a container.
+* Rows are nested in a container.
+* Columns are nested in a row.
+* There are a maximum of 12 columns.
+
+This image represents one container with seven rows. Each row has different sized elements that span a different number of columns.
+
+![Bootstrap Container](/assets/images/module3/Week5/BootstrapGrid.png)
+
+* The first row has 12 elements, each element spans 1 column.
+* The second row has 6 elements, each element spans 2 columns.
+* The third row has 4 elements, each element spans 3 columns.
+* The fourth row has 3 elements, each element spans 4 columns.
+* The fifth row has 2 elements, each element spans 6 columns.
+* The sixth row also has 2 elements, but each element span a different number of columns (4 and 8, respectively).
+* The last row has 1 element that spans all 12 columns.
+
+_Note:_ You can have exactly 12 columns, or less than 12 columns (the unused columns will appear as empty space). However, you cannot have more than 12 columns.
+
+## Responsiveness
+
+* view the [website](https://launchbootstrap.launch-team.repl.co/)
+* view the [code](https://replit.com/@launch-team/LaunchBootstrap#index.html) (and fork if you want)
+
+This is where the "responsive" part happens. Here is a screenshot of our example webpage as viewed on my desktop, in all its full-screen glory:
+
+![Website Desktop](/assets/images/module3/Week5/OlympicDesktop.png)
+
+Here is the same webpage, viewed on my tablet in portrait mode:
+![Website Tablet](/assets/images/module3/Week5/OlympicTablet.png)
+
+Some adjustments have been made for the smaller screen, but it remains unchanged for the most part.
+
+Same webpage, this time on my phone:
+![Website Tablet](/assets/images/module3/Week5/OlympicPhone.png)
+
+Whoa, big difference. But it would have been really difficult to read the text and view the images if they were all still side by side. I have to scroll to view the information for Paris (not to mention Beijing), but the experience is much better.
+
+## Classes
+
+The beauty of Bootstrap is that you attach classes to HTML tags, and the magic just... happens.
```

Comment:
I think we should add some more information about the 'magic'.  Maybe we can make it more clear that we are attaching classes that correspond with pre-existing css styling targets in bootstrap.

Reply:
you probably tell kids that Santa Claus doesn't exist, don't you? 😂 j/k